node-graph/gcore/src/vector/style.rs

```diff
@@ -14,7 +14,7 @@ const OPACITY_PRECISION: usize = 3;
 
 fn format_opacity(name: &str, opacity: f32) -> String {
 	if (opacity - 1.).abs() > 10_f32.powi(-(OPACITY_PRECISION as i32)) {
-		format!(r#" {}-opacity="{:.precision$}""#, name, opacity, precision = OPACITY_PRECISION)
+		format!(r#" {name}-opacity="{opacity:.OPACITY_PRECISION$}""#)
```

Comment:
Just commenting on this one so I can double-check it's correct.

Reply:
This one got auto-fixed with clippy, but yes, i believe you can inline the width too (I wrote the clippy lint for it, so I would take it with a grain of salt :D )

```
cargo clippy --workspace --allow-dirty --fix --benches --tests --bins -- -A clippy::all -W clippy::uninlined_format_args
```

Reply:
I'm just unfamiliar with that Rust string formatter syntax. But it sounds like you're the expert on that if you wrote the lint for it!

Reply:
famous last words :) I did look into it at some point, learning it in depth, but not the most fun topic to learn tbh :)